Resources/doc/reference/form_types.rst

```diff
@@ -487,12 +487,12 @@ Example with ``doctrine_orm_date_range`` filter:
         // ...
     }
 
-sonata_type_color_picker
+sonata_type_color
```

Comment:
Not documenting the deprecated type might be good, we shouldn't encourage its usage.